app/v1/dist/index.html

```diff
@@ -119,13 +117,13 @@
               cube({ size: 1, center: true }).translate([0.5, 0.5, 0.5]).as('b'),
               cube({ size: 1, center: true }).translate([0.5, 0, 0]).as('c').material('metal'))
 
-writePaths({ path: 'assembly' },
-           x.toPaths({ tags: ['a'] }),
-           x.toPaths({ tags: ['b'] }),
-           x.toPaths({ tags: ['c'] }))
+writeStl({ path: 'assembly' },
+         x.toSolid({ tags: ['a'] }),
```

Comment:
I like this notation better, it makes more sense to me at a glance

Reply:
Cool.

Although I hope that the x.toSolid() stuff can be made mostly invisible to end-users at some point by adding some nice API support. :)